client/landing/jetpack-cloud/sections/backups/components/upsell/index.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import React, { FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { Button, Card } from '@automattic/components';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	siteSlug: string;
+}
+
+const JetpackCloudBackupUpsell: FunctionComponent< Props > = ( { siteSlug } ) => {
+	const translate = useTranslate();
+
+	const jetpackBackupLink = `https://jetpack.com/upgrade/backup/?site=${ siteSlug }`;
+
+	return (
+		<Card className="upsell">
+			<div className="upsell__header">
+				<img
```

Comment:
This image appears broken for me. Should we include the SVG as a component in the PR?

![Screen Shot 2020-04-15 at 10 31 27 AM](https://user-images.githubusercontent.com/5528445/79349465-3e4c3580-7f04-11ea-9394-14c20f4eb9f3.png)

Reply:
you probably need to reboot jetpack cloud. It doesn't seem to pick up new static files with hot updates.

Reply:
Yes, that's right, I had to restart JP Cloud to view the icon